automation/repeated_test.sh

```diff
@@ -225,10 +240,12 @@ if [[ "${rwofs_sc}" == "local" ]]; then
 fi
 
 label_filter="(flake-check)||(${label_filter})"
-ginko_params="$ginko_params -no-color -succinct --label-filter="${label_filter}" -randomize-all"
-for test_file in $(echo "${NEW_TESTS}" | tr '|' '\n'); do
-    ginko_params+=" -focus-file=${test_file}"
-done
+ginko_params="$ginko_params -no-color -succinct --label-filter=${label_filter} -randomize-all"
+if [[ -n ${NEW_TESTS} ]]; then
+    IFS=$'\n'; for test_name in $(jq -r '.[]' "${NEW_TESTS}"); do
+        ginko_params+=" -focus='${test_name// /\\s}'"
```

Comment:
The separator is `\s`?? They don't do comas or spaces? Either way, please add a comment to explain the obscure substitution.

Reply:
Good point - it seems that this was only the case for `--focus-file`, it seems to work with `--focus` :+1: